src/ontology/mondo-edit.obo

```diff
@@ -322737,7 +322740,7 @@ xref: Orphanet:397959 {source="MONDO:equivalentTo", source="OMIM:615387"}
 xref: UMLS:C3809332 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:815662"}
 is_a: MONDO:0018814 {source="Orphanet:397959", source="https://orcid.org/0000-0001-5208-3432"} ! non-SCID combined immunodeficiency
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0014160 {source="MONDO:CLINGEN"}
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12029 {source="MONDO:mim2gene_medgen", source="OMIM:615387"} ! TRAC
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12029 {source="MONDO:mim2gene_medgen"} ! TRAC
```

Comment:
Same question as above, I still see this entry in omim. Can you please confirm that it is removed in the omim files?

Reply:
It looks like `relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12029 {source="MONDO:mim2gene_medgen", source="OMIM:615387"} ! TRAC` should have been maintained.